src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -947,7 +947,7 @@ window.qBittorrent.DynamicTable = (function() {
                         state = "uploading";
                         break;
                     case "pausedDL":
-                        state = "media-playback-pause";
+                        state = "torrent-stop";
```

Comment:
I can tell you that whatever this PR is doing doesn't fix what issue# 17357 brought up.
Look at line 947 and line 974. There's no `icons/uploading.svg`.

Reply:
I am working on it. First solution didn't work out.

Reply:
Can you check now?